src/reviewRenderer/renderers/interactions/MediaInteraction.js

```diff
@@ -0,0 +1,138 @@
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+/**
+ * @author Ansul Sharma <ansultaotesting.com>
+ */
+import $ from 'jquery';
+import _ from 'lodash';
+import containerHelper from 'taoQtiItem/qtiCommonRenderer/helpers/container';
+import mediaInteraction from 'taoQtiItem/qtiCommonRenderer/renderers/interactions/MediaInteraction';
+import mediaplayer from 'ui/mediaplayer';
+
+const defaults = {
+    type: 'video/mp4',
+    video: {
+        height: 270
+    }
+};
+
+/**
+ * Init rendering, called after template injected into the DOM
+ * All options are listed in the QTI v2.1 information model:
+ * http://www.imsglobal.org/question/qtiv2p1/imsqti_infov2p1.html#element10391
+ *
+ * @param {object} interaction
+ * @returns {Promise<any>}
+ */
+const render = function render(interaction) {
+    return new Promise((resolve) => {
+        const $container = containerHelper.get(interaction);
+        const media = interaction.object;
+        const $item = $container.parents('.qti-item');
+        const url = media.attr('data') || '';
+
+        /**
+         * Resize video player elements to fit container size
+         * @param {Object} mediaElement - player instance
+         * @param {jQueryElement} $container   - container element to adapt
+         */
+        const resize = _.debounce(() => {
+            let width, height;
+            if (interaction.mediaElement) {
+                height = $container.find('.media-container').height();
+                width = $container.find('.media-container').width();
+
+                interaction.mediaElement.resize(width, height);
+            }
```

Comment:
```suggestion
            if ( !interaction.mediaElement ) { 
                return;
            }
                const height = $container.find('.media-container').height();
                const width = $container.find('.media-container').width();

                interaction.mediaElement.resize(width, height);
            }
```

I would also suggest caching `const $mediaContainer = $container.find('.media-container')` selector.